parquet-column/src/main/java/org/apache/parquet/column/statistics/LongStatistics.java

```diff
@@ -62,23 +84,19 @@ public void setMinMaxFromBytes(byte[] minBytes, byte[] maxBytes) {
   }
 
   @Override
-  public boolean isSmallerThan(long size) {
-    return !hasNonNullValue() || (16 < size);
+  String toString(Long value) {
+    // TODO: implement unsigned int as required
```

Comment:
I think these should be finished before merging this.

Reply:
Created separate JIRA for it.